api/apps/articles/model.coffee

```diff
@@ -65,6 +65,7 @@ schema = (->
   featured_artwork_ids: @array().items(@objectId()).allow(null)
   fair_id: @objectId().allow('', null)
   partner_ids: @array().items(@objectId()).allow(null)
+  auction_id: @objectId().allow('', null)
```

Comment:
Good point... I'm not strongly opinionated on that naming. I like to optimistically think we'll some day refactor auctions to a first class thing, so I'm leaning towards `auction_id`.

For now yeah single auction is fine. We may need to make this a many to many later down the line... but I'll also probably have to address that with fairs—I'll cross that bridge when we get to it.